src/ape/api/accounts.py

```diff
@@ -134,6 +137,7 @@ def deploy(self, contract_type: ContractType, *args, **kwargs) -> ContractInstan
 class AccountContainerAPI:
     data_folder: Path
     account_type: Type[AccountAPI]
+    config_manager: "ConfigManager"
```

Comment:
Any container that is for testing needs to know the configuration of the test config